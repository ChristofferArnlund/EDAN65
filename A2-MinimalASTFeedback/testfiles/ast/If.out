Program
  List
    Func
      IdDecl ID="a"
        Opt
      List
      StmtBlock
        List
          IdDecl ID="x"
            Opt
          IdDecl ID="y"
            Opt
          If
            Equals
              IdUse ID="x"
              Numeral NUMERAL="5"
            StmtBlock
              List
                IdDecl ID="z"
                  Opt
            Opt
          If
            Le
              IdUse ID="y"
              Numeral NUMERAL="5"
            StmtBlock
              List
            Opt
              StmtBlock
                List
