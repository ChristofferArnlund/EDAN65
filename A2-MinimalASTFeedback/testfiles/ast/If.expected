Program
  List
    Func
      Name ID="a"
      List
      List
        IdDecl
          Name ID="x"
          Opt
        IdDecl
          Name ID="y"
          Opt
        If
          Equals
            Name ID="x"
            Numeral NUMERAL="5"
          List
            IdDecl
              Name ID="z"
              Opt
          List
        If
          Le
            Name ID="y"
            Numeral NUMERAL="5"
          List
          List
